src/DoctrineORMModule/Module.php

```diff
@@ -139,7 +138,14 @@ public function initializeConsole(EventInterface $event)
         $entityManager = $serviceLocator->get('doctrine.entitymanager.orm_default');
         $helperSet     = $cli->getHelperSet();
 
-        $helperSet->set(new QuestionHelper(), 'question');
+        if (class_exists('Symfony\Component\Console\Helper\QuestionHelper')) {
+            $helperSet->set(new \Symfony\Component\Console\Helper\QuestionHelper(), 'question');
+        }
+
+        if (class_exists('Symfony\Component\Console\Helper\DialogHelper')) {
+            $helperSet->set(new \Symfony\Component\Console\Helper\DialogHelper(), 'dialog');
```

Comment:
you should set the DialogHelper only when the QuestionHelper is not available, otherwise you will still trigger deprecation notices here